### ISSUE TITLE
website: icon a11y docs

### ISSUE THREAD

**Issue by coryrylan** [vmware/clarity#5568]
## Describe the bug

The website docs for icon a11y are missing for the new cds-icon recommendations

## How to reproduce

Previous content:
 https://clarity.design/foundation/icons/accessibility/

## Expected behavior

Recommend a11y guidelines for cds-icon 
https://clarity.design/storybook/core/?path=/story/components-icon--page

## Versions
@cds/core 5.0
